app/actions/remote/entry/notification.ts

```diff
@@ -99,7 +111,7 @@ export async function pushNotificationEntry(serverUrl: string, notification: Not
     const {id: currentUserId, locale: currentUserLocale} = (await getCurrentUser(operator.database))!;
     const {config, license} = await getCommonSystemValues(operator.database);
 
-    await deferredAppEntryActions(serverUrl, lastDisconnectedAt, currentUserId, currentUserLocale, prefData.preferences, config, license, teamData, chData, selectedTeamId, switchedToChannel ? selectedChannelId : undefined);
+    await deferredAppEntryActions(serverUrl, lastDisconnectedAt, currentUserId, currentUserLocale, prefData.preferences, config, license, teamData, chData, selectedTeamId, switchedToScreen ? selectedChannelId : undefined);
```

Comment:
this is skipping fetching posts to of the channel even if we opened a thread instead of the channel.